### PR TITLE
doc: Update WaaG GSG to use ACRNGT GOP to install Windows

### PR DESCRIPTION
**Pull request by chejianj**
Signed-off-by: Jian Jun Chen <jian.jun.chen@intel.com>

### CODE REVIEW

**Comment by NanlinXie** [comment]
Hold on until verification pass.

File: doc/tutorials/using_windows_as_uos.rst [hunk 1]
```diff
@@ -313,90 +313,83 @@ Run these commands on the Service VM::
...
-   Or used Oracle driver
-   - Change ``-s 9,ahci,cd:./winvirtio.iso`` to your path to the winvirtio iso.
+#. Add the following lines **before** ``acrn-dm`` command line in ``install_win.sh``. It is used
+   to passthrough USB to WaaG, by which both mouse and keyboard can be used during Windows installation::
+
+      echo "8086 9d2f" > /sys/bus/pci/drivers/pci-stub/new_id
+      echo "0000:00:14.0" > /sys/bus/pci/devices/0000:00:14.0/driver/unbind
+      echo "0000:00:14.0" > /sys/bus/pci/drivers/pci-stub/bind
+
+   .. note:: You may need to change the bdf and vid/pid of the USB controller in above command.
```
Comment (gvancuts) [inline doc/tutorials/using_windows_as_uos.rst]:
```suggestion
   .. note:: You may need to change the BDF and VID/PID of the USB controller in above command to match those of your platform.
```

File: doc/tutorials/using_windows_as_uos.rst [hunk 2]
```diff
@@ -313,90 +313,83 @@ Run these commands on the Service VM::
...
+      You can use ``lspci`` and ``lspci -n`` to get this information.
 
-#. Run ``install_vga.sh`` and connect to the Windows guest using a vnc client.::
+#. Edit the ``acrn-dm`` command line in ``install_win.sh`` as follows:
+
+   - Change ``-s 3,virtio-blk,./win10-ltsc-virtio.img`` to your path to the Windows 10 image.
+   - Add ``-s 6,passthru,0/14/0``.
```
Comment (gvancuts) [inline doc/tutorials/using_windows_as_uos.rst]: Does that need to be adjusted in case the BDF/VID/PID is different?

**Comment by acrnsi** [comment]
OK to verify

File: doc/tutorials/using_windows_as_uos.rst (hunk 2 above)
Comment (chejianj) [reply doc/tutorials/using_windows_as_uos.rst]: fix in new PR. Please check

**Comment by huyuan3x** [comment]
Install Windows successfully on ACRN with 31300 version clearlinux server os

**Comment by huyuan3x** [comment]
use ACRNGT GOP to install Windows successfully in KBLNUC

**Pull request merged**